app/src/main/java/com/duckduckgo/app/onboarding/ui/page/extendedonboarding/ExtendedOnboardingFeatureToggles.kt

```diff
@@ -31,14 +39,73 @@ interface ExtendedOnboardingFeatureToggles {
     fun self(): Toggle
 
     @Toggle.DefaultValue(false)
-    @Experiment
     fun noBrowserCtas(): Toggle
 
     @Toggle.DefaultValue(false)
-    @Experiment
     fun privacyProCta(): Toggle
 
     @Toggle.DefaultValue(false)
     @Experiment
     fun highlights(): Toggle
+
+    @Toggle.DefaultValue(false)
+    fun testPrivacyProOnboardingCopyNov24(): Toggle
+
+    enum class Cohorts(override val cohortName: String) : CohortName {
+        CONTROL("control"),
+        PROTECTION("protection"),
+        PIR("pir"),
+        VPN("vpn"),
+    }
+
+    companion object {
+        const val EXPERIMENT_PREFIX = "test"
+    }
+}
+
+internal suspend fun ExtendedOnboardingPixelsPlugin.testPrivacyProOnboardingShownMetricPixel(): MetricsPixel? {
+    return this.getMetrics().firstOrNull { it.metric == "dialogShown" }
+}
+
+internal suspend fun ExtendedOnboardingPixelsPlugin.testPrivacyProOnboardingPrimaryButtonMetricPixel(): MetricsPixel? {
+    return this.getMetrics().firstOrNull { it.metric == "primaryButtonSelected" }
+}
+
+internal suspend fun ExtendedOnboardingPixelsPlugin.testPrivacyProOnboardingSecondaryButtonMetricPixel(): MetricsPixel? {
+    return this.getMetrics().firstOrNull { it.metric == "secondaryButtonSelected" }
+}
+
+suspend fun FeatureTogglesInventory.onboardingExperiments(): Toggle? {
+    return this.getAllTogglesForParent("extendedOnboarding").firstOrNull {
+        it.featureName().name.startsWith(EXPERIMENT_PREFIX) && it.isEnabled()
+    }
+}
+
+@ContributesMultibinding(AppScope::class)
+class ExtendedOnboardingPixelsPlugin @Inject constructor(private val inventory: FeatureTogglesInventory) : MetricsPixelPlugin {
```

Comment:
Rather than injecting the inventory you can just inject `ExtendedOnboardingFeatureToggles` and then use the toggle for the experiment.